src/apps/stellar/layout.py

```diff
@@ -0,0 +1,84 @@
+from trezor import ui, utils
+from trezor.messages import ButtonRequestType
+from trezor.ui.text import Text
+
+from apps.common.confirm import require_confirm, require_hold_to_confirm
+from apps.stellar import consts
+
+
+async def require_confirm_init(ctx, address: str, network_passphrase: str):
+    text = Text("Confirm Stellar", ui.ICON_SEND, icon_color=ui.GREEN)
+    text.normal("Initialize singing with")
+    text.mono(*split(address))
+    await require_confirm(ctx, text, ButtonRequestType.ConfirmOutput)
+    network = get_network_warning(network_passphrase)
+    if network:
+        text = Text("Confirm network", ui.ICON_CONFIRM, icon_color=ui.GREEN)
+        text.normal("Transaction is on")
+        text.bold(network)
+        await require_confirm(ctx, text, ButtonRequestType.ConfirmOutput)
+
+
+async def require_confirm_memo(ctx, memo_type: int, memo_text: str):
+    if memo_type == consts.MEMO_TYPE_TEXT:
+        title = "Memo (TEXT)"
+    elif memo_type == consts.MEMO_TYPE_ID:
+        title = "Memo (ID)"
+    elif memo_type == consts.MEMO_TYPE_HASH:
+        title = "Memo (HASH)"
+    elif memo_type == consts.MEMO_TYPE_RETURN:
+        title = "Memo (RETURN)"
+    else:  # MEMO_TYPE_NONE
+        title = "No memo set!"  # todo format this as ui.NORMAL not MONO
```

Comment:
Is this TODO still relevant?

Reply:
The problem was that it was quite ugly, because the text `Important: Many exchanges require a memo when depositing` gets formatted as MONO. I think the change would be easier now, because of the new API, so I'll look into it. Or we can leave it for the "UI overall check" as discussed. What do you think?

Reply:
Fixed.

Reply:
Thanks!